clipper_admin/clipper_admin/sample_config.yaml

```diff
@@ -0,0 +1,46 @@
+Model1:
+  obj_type: "model"
+  input_type: "integers"
+  version: "1.2.7"
+  image: "test-model:1"
+
+Application1:
+  obj_type: "application"
```

Comment:
As suggested by @simon-mo , let's define separate arrays called "Applications" and "Models". We can then remove the `obj_type` field from elements of these arrays because every entry under "Applications" will be an application object. i.e. Do the same thing for models and applications that you've done for links.